tests/migration/Core/V6_6/Migration1739355802FixVatHandlingTest.php

```diff
@@ -0,0 +1,73 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_6;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Migration\V6_6\Migration1739355802FixVatHandling;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+#[CoversClass(Migration1739355802FixVatHandling::class)]
+class Migration1739355802FixVatHandlingTest extends TestCase
+{
+    private Connection $connection;
+
+    protected function setUp(): void
+    {
+        $this->connection = KernelLifecycleManager::getConnection();
+    }
+
+    public function testUpdateAddsVatPatterns(): void
+    {
+        $this->removeVatPatterns();
+
+        $migration = new Migration1739355802FixVatHandling();
+        $migration->update($this->connection);
+
```

Comment:
```suggestion
        $migration = new Migration1739355802FixVatHandling();
        $migration->update($this->connection);
        $migration->update($this->connection);
```

I am sure, this is not an issue, but for the sake of completeness, please test, that your migration runs twice properly, as all Shopware migrations should do